crates/analytics/docs/clickhouse/scripts/sdk_events.sql

```diff
@@ -156,7 +157,7 @@ WHERE
     length(_error) > 0;
 
 CREATE MATERIALIZED VIEW sdk_events_audit_mv TO sdk_events_audit (
-    `payment_id` Nullable(String),
+    `payment_id` String,
```

Comment:
The queue (sdk_events_queue) has payment_id as Nullable(String), is it fine to just have String in the table and mv?

Reply:
There is a where clause which has the condition `payment_id` is not NULL in the mv, which resolves the query.